bluechi.spec.in

```diff
@@ -0,0 +1,230 @@
+Name:    bluechi
+Version: @VERSION@
+Release: @RELEASE@%{?dist}
+Summary: A systemd service controller for multi-nodes environments
+License: GPLv2+
+URL:     https://github.com/containers/bluechi
+Source0: %{url}/archive/v%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires: gcc
+BuildRequires: meson
+BuildRequires: systemd-devel
+BuildRequires: systemd-rpm-macros
+BuildRequires: golang-github-cpuguy83-md2man
+
+Requires: systemd
+Recommends: bluechi-selinux
+Provides: hirte
+Obsoletes: hirte < 0.6.0
+Provides: hirte = %{version}-%{release}
+
+%description
+BlueChi is a systemd service controller for multi-nodes environements with a
+predefined number of nodes and with a focus on highly regulated environment
+such as those requiring functional safety (for example in cars).
+This package contains the controller and command line tool.
+
+%post
+%systemd_post bluechi.service
+
+%preun
+%systemd_preun bluechi.service
+
+%postun
+%systemd_postun_with_restart bluechi.service
+
+%files
+%ghost %{_sysconfdir}/bluechi/bluechi.conf
+%dir %{_sysconfdir}/bluechi
+%dir %{_sysconfdir}/bluechi/bluechi.conf.d
+%doc README.md
+%doc README.developer.md
+%license LICENSE
+%{_bindir}/bluechi
+%{_datadir}/dbus-1/interfaces/org.containers.bluechi.Job.xml
+%{_datadir}/dbus-1/interfaces/org.containers.bluechi.Manager.xml
+%{_datadir}/dbus-1/interfaces/org.containers.bluechi.Monitor.xml
+%{_datadir}/dbus-1/interfaces/org.containers.bluechi.Node.xml
+%{_datadir}/dbus-1/system.d/org.containers.bluechi.conf
+%{_datadir}/bluechi/config/bluechi.conf
+%{_mandir}/man1/bluechi.*
+%{_mandir}/man5/bluechi.conf.*
+%{_sysconfdir}/bluechi/bluechi.conf.d/README.md
+%{_unitdir}/bluechi.service
+%{_unitdir}/bluechi.socket
+
+
+%package agent
+Summary:  BlueChi service controller agent
+Requires: systemd
+Recommends: bluechi-selinux
+
+%description agent
+BlueChi is a systemd service controller for multi-nodes environements with a
+predefined number of nodes and with a focus on highly regulated environment
+such as those requiring functional safety (for example in cars).
+This package contains the node agent.
+
+%post agent
+%systemd_post bluechi-agent.service
+
+%preun agent
+%systemd_preun bluechi-agent.service
+
+%postun agent
+%systemd_postun_with_restart bluechi-agent.service
+
+%files agent
+%ghost %{_sysconfdir}/bluechi/agent.conf
+%dir %{_sysconfdir}/bluechi
+%dir %{_sysconfdir}/bluechi/agent.conf.d
+%doc README.md
+%license LICENSE
+%{_bindir}/bluechi-agent
+%{_bindir}/bluechi-proxy
+%{_datadir}/dbus-1/system.d/org.containers.bluechi.Agent.conf
```

Comment:
`org.containers.hirte.` should be replaced with `io.github.eclipse-bluechi.bluechi` according to the doc and today's discussion